randnum.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
```

Comment:
Good job on this part! You could have done it a bit more efficiently, but you still did a good job implementing the assignment.